src/pathfinding_service/schema.sql

```diff
@@ -1,7 +1,7 @@
 CREATE TABLE iou (
     sender TEXT NOT NULL,
-    amount HEXINT NOT NULL,
-    expiration_block HEXINT NOT NULL,
+    amount HEX_INT NOT NULL,
```

Comment:
Mh, how did this work before?

Reply:
We were just storing the numbers as ints in the database. This would have failed once we got to values larger than MAXINT.